docs/rules/forbid-component-props.md

```diff
@@ -35,7 +35,7 @@ The following patterns are **not** considered warnings:
 
 ```js
 ...
-"react/forbid-component-props": [<enabled>, { "forbid": [<string>|<object>] }]
+"react/forbid-component-props": [<enabled>, { "forbid": [<string>|<object>], "message": <string> }]
```

Comment:
this seems like it would allow only one message for each "forbid" value; it seems like it. might make more sense to add a `message` property on the object form of a `forbid` value?

Reply:
@ljharb Thanks for taking a look. I've updated this to allow a separate message for each forbid option. Let me know what you think!